pkg/cmd/repo/create/http.go

```diff
@@ -236,6 +238,76 @@ func resolveOrganizationTeam(client *api.Client, hostname, orgName, teamSlug str
 	return &response, err
 }
 
+func listRepositoryTemplates(client *http.Client, hostname, owner string) (*list.RepositoryList, error) {
+	ownerConnection := "repositoryOwner(login: $owner)"
+
+	variables := map[string]interface{}{
+		"perPage": githubv4.Int(100),
+		"owner":   githubv4.String(owner),
+	}
+	inputs := []string{"$perPage:Int!", "$endCursor:String", "$owner:String!"}
+
+	type result struct {
+		RepositoryOwner struct {
+			Login        string
+			Repositories struct {
+				Nodes      []api.Repository
+				TotalCount int
+				PageInfo   struct {
+					HasNextPage bool
+					EndCursor   string
+				}
+			}
+		}
+	}
+
+	query := fmt.Sprintf(`query RepositoryList(%s) {
+		%s {
+			login
+			repositories(first: $perPage, after: $endCursor, ownerAffiliations: OWNER, orderBy: { field: PUSHED_AT, direction: DESC }) {
+				nodes{
+					id
+					name
+					isTemplate
+					pushedAt
```

Comment:
In the interest of reducing bandwidth, I'm curious whether we actually need `name` or `pushedAt`. They don't look used in the code and I don't think `pushedAt` is required for the `orderBy` to work?

Reply:
You are right, it works without having both `name` and `pushedAt`.

Reply:
Ah, I don't think it does actually. I hadn't noticed that `Name` is used to [present the prompter list](https://github.com/cli/cli/pull/7769/files#diff-1383afb03d1730b7f23e2da529e535aa1fc1c5ac500fc802899dafa66fcbb61eR799). Now we just get an empty list:

<img width="1066" alt="image" src="https://github.com/cli/cli/assets/1611510/bab3f7c1-6990-41f4-87e3-1582f9f5585f">

Reply:
Sorry I missed it. Thanks for noticing. I added name to the query.